cloudpickle/cloudpickle.py

```diff
@@ -123,6 +127,32 @@ def _lookup_class_or_track(class_tracker_id, class_def):
     return class_def
 
 
+def register_pickle_by_value(module):
+    """Register that the input module should be pickled by value."""
+    module_name = module.__name__ if inspect.ismodule(module) else module
+    _PICKLE_BY_VALUE_MODULES.add(module_name)
+
+
+def unregister_pickle_by_value(module):
+    """Unregister that the input module should be pickled by value."""
+    module_name = module.__name__ if inspect.ismodule(module) else module
+    _PICKLE_BY_VALUE_MODULES.remove(module_name)
+
+
+def is_registered_pickle_by_value(module, submodules=True):
```

Comment:
It does not seem like `submodules=False` used anywhere in `cloudpickle` logic right now. Can we just remove that argument @Samreay?